referenceframe/frame_test.go

```diff
@@ -157,10 +157,8 @@ func TestSerializationStatic(t *testing.T) {
 	f2if, err := f2Cfg.ParseConfig()
 	test.That(t, err, test.ShouldBeNil)
 
-	f2, err := f2if.ToStaticFrame("")
+	_, err = f2if.ToStaticFrame("")
 	test.That(t, err, test.ShouldBeNil)
-
-	test.That(t, f.AlmostEquals(f2), test.ShouldBeTrue)
```

Comment:
I'm in favor of getting rid of this function but I don't want to degrade our test quality.  So lets find a way to still test for this here

Reply:
I think the whole point of the function in the first place might have been for tests